action.yml

```diff
@@ -23,6 +23,16 @@ description: |
           ecr_repository: 'demo/example'
           python-version: 3.10
   ```
+
+  ## Locating Container Images
+
+  Python Docker images are located in AWS ECR, V-Prod. Their URI looks like the below snippet.
```

Comment:
Can you make it same as for other actions.

Reply:
changed to match others